mteb/tasks/Classification/eng/LegalBenchClassification.py

```diff
@@ -3026,152 +3026,6 @@ def dataset_transform(self):
         self.dataset = self.dataset.rename_column("answer", "label")
 
 
-class Diversity1LegalBenchClassification(AbsTaskClassification):
```

Comment:
I don't think we should remove these. Though I agree that they are very challenging tasks (and potentially problematically so).